### ISSUE TITLE
Playlist - Progress Report -- Inconsistent  ( Not Sure whether its correct or not)

### ISSUE THREAD

**Issue by anuragkanungo** [learningequality/ka-lite#2407]
Branch : nalanda-rct3
Expected Behaviour: It should show Playlist 3 only once and contain equivalent number of videos and exercises.
Current Behavious : Wrong Number of Playlist it shoud be 4 but shows playlist 3 twice with 10+17 videos and 6+9 exercies where as it should contains 2 videos and 1 exercise.

Steps to Reproduce:
1. Login as student ( assumed student account is in some group with playlists)
2. Go to playlist ( in this case was basic subtraction) watch videos and attempt exercies.
3. Go to My progress and there you can see that issue.

Attached Screenshots
![screenshot from 2014-09-24 22 52 25](https://cloud.githubusercontent.com/assets/4207886/4392624/80d2c1c4-4410-11e4-8b53-6d55e5e24301.png)
![screenshot from 2014-09-24 22 52 15](https://cloud.githubusercontent.com/assets/4207886/4392622/805659e0-4410-11e4-90d5-c3893a835f58.png)

**Comment by rtibbles** [comment]
Won't fix.
